src/options/introspect.rs

```diff
@@ -43,7 +43,12 @@ pub struct IntrospectOpts {
 }
 
 impl IntrospectOpts {
-    pub async fn exec_and_watch<F, G>(&self, exec_fn: F, output_opts: &OutputOpts) -> !
+    pub async fn exec_and_watch<F, G>(
+        &self,
+        exec_fn: F,
+        output_opts: &OutputOpts,
+        hide_output: bool,
```

Comment:
stating opinion only, no action requested: I try to avoid boolean arguments, but i'm also curious why `hide` isn't an `output_opts`

Reply:
agreed about boolean arguments, but this was the quickest way to get it done; I didn't want to put it into output_opts because the blast radius of it not working as expected felt big (with `arg(skip)` to hide it from the help, which means no default can be set; the compiler didn't complain, but I felt pretty sketched out by it and didn't want to spend time figuring out how to get confident in its use with other commands)